dnf/cli/commands/distrosync.py

```diff
@@ -28,7 +28,7 @@ class DistroSyncCommand(commands.Command):
     distro-synch command.
     """
 
-    aliases = ('distro-sync', 'distrosync', 'distribution-synchronization')
+    aliases = ('distro-sync', 'distrosync', 'distribution-synchronization', 'dup')
```

Comment:
`dup` is short for `distro-upgrade` / `distribution-upgrade` (from Zypper). I'd suggest adding the alias `dist-sync` or `dsync`. Also supporting the Zypper names would probably be a nice touch.